Gateway/Response/TransactionRefundHandler.php

```diff
@@ -64,9 +64,9 @@ public function handle(array $handlingSubject, array $response)
         $id = $this->localeDate->date()
             ->format('His');
         $id = ($id) ? $id : 1;
-        $chargeId = $orderPayment->getAdditionalInformation(self::CHARGE_ID);
+        $transactionId = $orderPayment->getAdditionalInformation(self::TRANSACTION_ID);
         $type = Transaction::TYPE_REFUND;
-        $orderPayment->setTransactionId("{$chargeId}-{$id}-{$type}");
+        $orderPayment->setTransactionId("{$transactionId}-{$id}-{$type}");
```

Comment:
Should we set the type here as well for us to know whether the transaction was processed using charges api or transaction api ? 🍹